tools/cli/adminFailoverCommands.go

```diff
@@ -273,6 +200,73 @@ func getRunID(c *cli.Context) string {
 	return ""
 }
 
+func failoverStart(c *cli.Context, params *startParams) {
+	validateStartParams(params)
+
+	targetCluster := params.targetCluster
+	sourceCluster := params.sourceCluster
+	batchFailoverSize := params.batchFailoverSize
+	batchFailoverWaitTimeInSeconds := params.batchFailoverWaitTimeInSeconds
+	workflowTimeout := time.Duration(params.failoverTimeout) * time.Second
+	domains := params.domains
+	drillWaitTime := time.Duration(params.drillWaitTime) * time.Second
+
+	client := getCadenceClient(c)
+	tcCtx, cancel := newContext(c)
+	defer cancel()
+
+	options := cclient.StartWorkflowOptions{
+		ID:                           failovermanager.WorkflowID,
+		WorkflowIDReusePolicy:        cclient.WorkflowIDReusePolicyAllowDuplicate,
+		TaskList:                     failovermanager.TaskListName,
+		ExecutionStartToCloseTimeout: workflowTimeout,
+		Memo: map[string]interface{}{
+			common.MemoKeyForOperator: getOperator(),
+		},
+	}
+	if params.drillWaitTime > 0 {
+		options.ID = failovermanager.DrillWorkflowID
+		options.CronSchedule = params.cron
+	} else {
+		// block if there is an on-going failover drill
+		if err := executePauseOrResume(c, failovermanager.DrillWorkflowID, true); err != nil {
```

Comment:
Looks like it's possible that one drill workflow may receive multiple pause signals? In that case how many resume signal is needed to unblock it?

Reply:
Added a step to clean up the pause channel after receiving a resume signal. This should dedup the pause signals.